sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/ingestion/models/ErrorAttachmentLog.java

```diff
@@ -79,6 +79,9 @@
      * @return ErrorAttachmentLog built attachment.
      */
     public static ErrorAttachmentLog attachmentWithText(String text, String fileName) {
+        if (text == null) {
```

Comment:
shouldn't that be an error?

Reply:
On iOS we can send attachments with null text, so we don't print error in this case.